src/main/target/common_pre.h

```diff
@@ -356,13 +356,11 @@
 #if (TARGET_FLASH_SIZE > 256)
 #define USE_AIRMODE_LPF
 #define USE_CANVAS
-#define USE_DASHBOARD
 #define USE_FRSKYOSD
 #define USE_GPS
 #define USE_GPS_NMEA
 #define USE_GPS_UBLOX
 #define USE_GPS_RESCUE
-#define USE_GYRO_DLPF_EXPERIMENTAL
```

Comment:
BMI270's FIFO mode needs this. See @etracer65 's comment

Reply:
This only applies to targets with less than 512MB of memory, which are they and do we still support them?